web/jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  resetMocks: true,
+  restoreMocks: true,
+  setupFilesAfterEnv: ['<rootDir>/test/setup-tests.js'],
+  testMatch: ['**/test/**/*-test.js'],
+  testPathIgnorePatterns: ['/node_modules/', '/test/support/', '.cache', '/.next/'],
+  testURL: 'http://localhost',
+  transform: {
+    '^.+\\.(js|jsx|ts|tsx)$': '<rootDir>/node_modules/babel-jest',
+  },
```

Comment:
Can you try something like this to get your imports to work?

```suggestion
  },
  "moduleNameMapper": {
    "src/(.*)$": "<rootDir>/src/$1",
   },
```

Reply:
That fixed it, thank you 🙂